sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/Azure.Monitor.OpenTelemetry.Exporter.Tests.csproj

```diff
@@ -4,11 +4,16 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\Integration.Tests\TestFramework\MockTransmitter.cs" Link="MockTransmitter.cs" />
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="xunit" />
     <PackageReference Include="xunit.runner.visualstudio" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
+    <PackageReference Include="OpenTelemetry.Extensions.AzureMonitor" VersionOverride="1.0.0-beta.2" />
```

Comment:
does the test project need this explicit reference?
i would expect this to come from the Exporter project